graphyte.py

```diff
@@ -147,6 +154,8 @@ def send_socket(self, message):
             self.send_message(message)
         except Exception as error:
             logger.error('error sending message {!r}: {}'.format(message, error))
+            if self.raise_send_errors:
+                raise
```

Comment:
I think this would be better before the `logger.error()` call: if we're raising, that failure will blow up and go somewhere, so we don't need to duplicate the logging.